api/object.go

```diff
@@ -131,31 +131,115 @@ type (
 		MimeType string `json:"mimeType"`
 		ETag     string `json:"eTag"`
 	}
+
 	CopyObjectOptions struct {
 		MimeType string `json:"mimeType"`
 	}
+
 	DeleteObjectOptions struct {
 		Batch bool `json:"batch"`
 	}
+
+	DownloadObjectOptions struct {
```

Comment:
The `Options` types are only options for the client right? They don't get serialised? So we don't need json tags on them right?